pkg/apis/networking/types.go

```diff
@@ -90,6 +90,25 @@ type NetworkPolicyPort struct {
 	Port *intstr.IntOrString
 }
 
+// CIDR describes a Network CIDR to select
+type CIDR string
+
+// Except is a slice of CIDRs to not include within an IpBlockRule
+type Except []string
```

Comment:
You're not using either the `CIDR` type or the `Except` type. (IpBlockRule just uses `string` and `[]string` directly. And I feel like that's correct.)